### PR TITLE
Added landing page for rails and sinatra

### PR DESCRIPTION
**Pull request by Prakriti-nith**
Rails landing page:
landing_page_rails

Sinatra landing page:
landing_page_sinatra

### CODE REVIEW

**Comment by Prakriti-nith** [comment]
@Shekharrajak can you please review this PR?

**Review by Shekharrajak — COMMENTED** [review]
I am expecting better styling. It look like a list, isn't it? Why not use daru vector or dataframe and display it here using daru-view table ?

**Comment by Shekharrajak** [comment]
Let me know if you can't update it. @Prakriti-nith

**Comment by Prakriti-nith** [comment]
landing page of rails app now:
landing_page_rails

File: demo_rails/app/views/layouts/layout.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (Shekharrajak) [inline demo_rails/app/views/layouts/layout.html.erb]: I believe this whole page has to be removed now.

File: demo_sinatra/views/layout.erb [hunk 2]
```diff
@@ -20,7 +20,39 @@
 
     <p>
```
Comment (Shekharrajak) [inline demo_sinatra/views/layout.erb]: Need to update here as well

**Review by Shekharrajak — COMMENTED** [review]
So now you are using google chart datatable in landing page for all  available examples. It is looking good. I have commented few things, please check.

**Comment by Prakriti-nith** [comment]
Landing page of sinatra now:
sinatra_landing_page

File: demo_sinatra/app.rb [hunk 3]
```diff
@@ -61,6 +62,38 @@
   erb :formatters, :layout => :googlecharts_layout
 end
 
+def front_page
+  @links = Daru::DataFrame.new({
+    Examples:  [
```
Comment (Shekharrajak) [inline demo_sinatra/app.rb:67]: To have neat code, we should write these kinds long of list somewhere else and use it here. That would be easy to understand about the dataframe  parameter.

**Review by Shekharrajak — APPROVED** [review]
For now it is good to go. Thanks @Prakriti-nith

**Pull request merged**